Add support for Symfony 7.0

jerowork opened the pull request:
# Description
Update `composer.json` to support Symfony `^7.0`.

jerowork committed:
Add support for Symfony ^7.0

ruudk commented:
@Yozhef Would be great to have this merged and tagged.

Yozhef commented on composer.json:
```diff
@@ -17,13 +17,13 @@
-        "symfony/browser-kit":  "^4.4|^5.0|^6.0",
-        "symfony/dom-crawler":  "^4.4|^5.0|^6.0"
+        "symfony/browser-kit":  "^4.4|^5.0|^6.0|^7.0",
+        "symfony/dom-crawler":  "^4.4|^5.0|^6.0|^7.0"
```
Please add CI Process to run Symfony components version 7.0

jerowork replied:
Same here, added 7.0 to the GH actions 👍

Yozhef replied:
I'm sorry but CI has not started

jerowork replied:
Its because the test workflow has been disabled due to 60 days of inactivity. You can enable it again in the GH actions tab
<img width="979" alt="Scherm­afbeelding 2024-02-06 om 14 20 26" src="https://github.com/FriendsOfBehat/MinkBrowserKitDriver/assets/4119451/9c152649-03dc-45dd-b066-2b7265cc6ffd">

Yozhef replied:
Ready )

jerowork committed:
Add SF 7.0 to GH actions

Excluding all PHP versions lower than 8.2, as Symfony 7.0 minimum requirement
is PHP 8.2.

Yozhef commented:
@ruudk @jerowork thank you very much for the contribution and development - sorry for taking so long to watch
